test/AutoDiff/array.swift

```diff
@@ -27,19 +28,19 @@ ArrayAutodiffTests.test("ArraySubscript") {
     gradient(at: [2, 3, 4, 5, 6, 7], in: sumFirstThree))
 }
 
-ArrayAutodiffTests.test("ArrayLiteral") {
-  func twoElementLiteral(_ x: Float, _ y: Float) -> [Float] {
+ArrayAutoDiffTests.test("ArrayLiteral") {
+  func twoElementLiteral(_ x: Tracked<Float>, _ y: Tracked<Float>) -> [Tracked<Float>] {
```

Comment:
We should migrate all `Float`-only tests to use `Tracked<Float>` (our nontrivial type) to prevent these bugs from happening.